recipes/tyssue/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "tyssue" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "3787cd9bb9292f9ecb8d5a91db8706bfe95845f011b37f2890ece425270aae8d" %}
+
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/DamCB/tyssue/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - pandas
+    - pytables
+    - jupyter
+    - notebook
+test:
+  imports:
+    - tyssue
+about:
+  home: https://github.com/DamCB/tyssue
+  license: MPLv2.0
+  summary: 'A tissue simulation library'
+  description: |
+    tyssue provides data structures and algorihms to model epithelium
+    in 2D and 3D for python 3.4+
```

Comment:
If this package is really only python 3.4+, please use the appropriate "skip" in the build:

http://conda.pydata.org/docs/building/meta-yaml.html#skipping-builds